package.json

```diff
@@ -143,13 +144,73 @@
                 "category": "Azure Resource Manager Tools",
                 "title": "Reload Cached Schemas",
                 "command": "azurerm-vscode-tools.reloadSchemas"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort template",
+                "command": "azurerm-vscode-tools.sortTemplate"
+            },
+            {
+                "category": "Azure Resource Manager Tools",
+                "title": "Sort functions",
```

Comment:
If we keep all these commands, they need to all use "title casing" for the title, meaning all words are capitalized (except for stuff like "a", "the".

Reply:
Fixed.